### PR TITLE
create page 10 facts

### PR DESCRIPTION
**Pull request by Kiril99**
done HW2

### CODE REVIEW

File: HW2/Responsive/src/index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            <h2>The dark side of the moon is a myth.</h2>
+            <p>
+            In reality both sides of the Moon see the same amount of sunlight
+            however only one face of the Moon is ever seen from Earth. This is because
+            the Moon rotates around on its own axis in exactly the same time it takes
+            to orbit the Earth, meaning the same side is always facing the Earth.
+            The side facing away from Earth has only been seen by the human eye from spacecraft.
+            </p>
+          </div>
+          <img src="../assets/dark-side.png" alt="">
```
Comment (vakal237078) [inline HW2/Responsive/src/index.html]: Add 'alt' for all images

File: HW2/Responsive/src/index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Responsive</title>
+</head>
+<body>
+  <!-- Your code goes here -->
+  <section class="container">
+    <header>
+      <ul>
```
Comment (Davydovychyurii) [inline HW2/Responsive/src/index.html]: use the tag nat

File: HW2/Responsive/src/index.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+              however only one face of the Moon is ever seen from Earth. This is because
+              the Moon rotates around on its own axis in exactly the same time it takes
+              to orbit the Earth, meaning the same side is always facing the Earth.
+              The side facing away from Earth has only been seen by the human eye from spacecraft.
+            </p>
+          </div>
+          <img src="../assets/moon-station.png" alt="">
+        </div>
+
+        <div class="text-img reverse-box">
```
Comment (Davydovychyurii) [inline HW2/Responsive/src/index.html]: use semantic tags

File: HW2/Responsive/src/style.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
+/*
+ * Feel free to change name of colors if you need it
```
Comment (Davydovychyurii) [inline HW2/Responsive/src/style.css:2]: You need this commented line of code? If no, please remove this line of code

File: HW2/Responsive/src/index.html (hunk 2 above)
Comment (Davydovychyurii) [inline HW2/Responsive/src/index.html:14]: use the tag nav

File: HW2/Responsive/src/index.html [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  <link rel="stylesheet" href="style.css">
+  <title>Responsive</title>
+</head>
+<body>
+  <!-- Your code goes here -->
+  <section class="container">
+    <header>
+      <ul>
+        <a href=""><li>The Moon</li></a>
+        <a href=""><li>The Sun</li></a>
```
Comment (Davydovychyurii) [inline HW2/Responsive/src/index.html:16]: тег а потрібно обгортати в тег li, а не на навпаки.

File: HW2/Responsive/src/index.html (hunk 1 above)
Comment (Kiril99) [reply HW2/Responsive/src/index.html]: +

File: HW2/Responsive/src/index.html (hunk 2 above)
Comment (Kiril99) [reply HW2/Responsive/src/index.html:14]: +

File: HW2/Responsive/src/index.html (hunk 5 above)
Comment (Kiril99) [reply HW2/Responsive/src/index.html:16]: +

Comment (Davydovychyurii) [reply HW2/Responsive/src/index.html:16]: you did not correct the comment

**Pull request merged**